tests/system/action/motion/test_create.py

```diff
@@ -401,3 +434,98 @@ def test_create_permission_lead_motion(self) -> None:
             },
         )
         self.assert_status_code(response, 200)
+        self.assert_model_exists("motion/4", {"block_id": 13, "category_id": 12})
+
+    def test_create_permission_amendment_2(self) -> None:
```

Comment:
Rename: `test_create_amendment_non_admin` or so.

Reply:
Please add a block and category to the lead motion, so it can be asserted, that these block/category are also taken for the amendment.